libs/shared/src/utils.ts

```diff
@@ -64,8 +64,8 @@ export async function getFileSizeBytes(url: string): Promise<number> {
     // Range header is to prevent it from reading any bytes from the GET request because we only want the headers.
     const response = await fetch(url, { headers: { Range: 'bytes=0-0' } });
     if (!response) return 0;
-    const contentRange = response.headers.get('content-range');
-    return contentRange ? parseInt(contentRange.split('/')[1], 10) : 0;
+    const contentLength = response.headers.get('content-length');
```

Comment:
So I don't think this works correctly. We have a test which apparently is not being run on the CI in checkIconSize.ts. This request actually doesn't have a content-length even though it is an uploaded image. It seems the response content-length header is set to 1 for whatever reason, although the image size is much bigger than 1.

The corresponding image:
https://commonwealth-uploads.s3.us-east-2.amazonaws.com/bebbda6b-6b10-4cbd-8839-4fa8d2a0b266.jpg

The response headers:
<img width="1040" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/14794654/709559cc-757c-43e9-8bc4-8f2eb477d860">